src/Xamarin.Android.Build.Tasks/Utilities/JavaResourceParser.cs

```diff
@@ -98,8 +102,9 @@ public JavaResourceParser ()
 					}),
 			Parse (@"^        public static final int\[\] ([^ =]+) = {",
 					(m, app, g, map) => {
+						if (g == null) return null;
 						var name = ((CodeTypeDeclaration) g.Members [g.Members.Count-1]).Name;
-						var f = new CodeMemberField (typeof (int[]), ResourceIdentifier.GetResourceName (name, m.Groups[1].Value, map, Log)) {
+						var f = new CodeMemberField (typeof (int[]), ResourceIdentifier.GetResourceName (name, m.Groups[1].Value, map, Log!)) {
```

Comment:
Tell me the compiler error if you remove this `!`, if there is no error can we just remove the `!`.